ScienceDirect.js

```diff
@@ -278,6 +278,10 @@ function processRIS(doc, text) {
 		}
 		if (item.ISBN && !ZU.cleanISBN(item.ISBN)) delete item.ISBN;
 		if (item.ISSN && !ZU.cleanISSN(item.ISSN)) delete item.ISSN;
+		
+		if (item.url && item.url.substr(0,2) == "//") {
+			item.url = "http:" + item.url;
```

Comment:
When I am entering `//www.sciencedirect.com/science/article/pii/B9780857095138500182` in my browser then it will be redirected to the `http` version. We can also switch to `https` if you prefer. Do you?